model/Export/AbstractQTIItemExporter.php

```diff
@@ -316,4 +326,26 @@ protected function getMetadataExporter()
         }
         return $this->metadataExporter;
     }
+
+    private function copySharedStimulusCssFiles(string $link, string $basePath): string
+    {
+        $asset = new core_kernel_classes_Resource(UriHelper::decode($link));
+        $baseName = 'assets/' . $this->getSharedStimulusCSSExporter()->getResourceLink($asset);
+        $this->getSharedStimulusCSSExporter()->pack(
+            $asset,
+            $this->getZip(),
+            $basePath . '/' . dirname($baseName)
+        );
+
+        return $baseName;
+    }
+
+    private function getSharedStimulusCSSExporter(): SharedStimulusCSSExporter
+    {
+        if (!isset($this->sharedStimulusCSSExporter)) {
+            $this->sharedStimulusCSSExporter = $this->getServiceManager()->get(SharedStimulusCSSExporter::class);
```

Comment:
Why you using `getServiceManager` to retrieve service instead of `ServiceLocator`?

Reply:
We need to leave ServiceManager call as the class is not ConfigurableService.